daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -1744,12 +1784,51 @@ class RazerBasiliskV3Pro35KWireless(RazerBasiliskV3Pro35KWired):
     USB_PID = 0x00CD
 
 
+class RazerBasiliskV3Pro35KPhantomGreenEditionWired(RazerBasiliskV3Pro35KWired):
+    """
+    Class for the Razer Basilisk V3 Pro 35K Phantom Green Edition (Wired)
+    """
+    USB_PID = 0x00D6
```

Comment:
plus `USB_VID = 0x1532`

Reply:
Yeah, and I just fixed that :D 09ec36ecaa34dde69d6d55bc13525cd3c5b0170e

Reply:
Fair! Will change that!

Reply:
Done! Just inherits from __RazerDevice now